dashboard/test/ui/runner.rb

```diff
@@ -474,6 +474,10 @@ def how_many_reruns?(test_run_string)
     arguments += " --format rerun --out #{rerun_filename}"
   end
 
+  if ENV['CI']
+    arguments += " --format junit --out $CIRCLE_TEST_REPORTS/cucumber/#{test_run_string}.xml"
+  end
```

Comment:
It might seem like overkill, but I'd recommend a comment on this block explaining that we're exporting _additional_ logs in JUnit XML format to improve reporting on Circle CI, and linking to https://circleci.com/docs/test-metadata/#cucumber